frontend/src/types/SelectInputTypes.ts

```diff
@@ -0,0 +1,10 @@
+import { OptionBase } from "chakra-react-select";
+
+export interface Option extends OptionBase {
+  value: string | boolean;
+  label: string;
+}
+
+export interface MultiOption extends Option {
```

Comment:
should this also have the label?

Reply:
It extends option so it inherits the label